extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -0,0 +1,319 @@
+    <bento-base-carousel snap-align="center" visible-count="1.2" loop>
+      <img src="server.com/static/inline-examples/images/image1.jpg">
+      <img src="server.com/static/inline-examples/images/image2.jpg">
+      <img src="server.com/static/inline-examples/images/image3.jpg">
+    </bento-base-carousel>
+    <bento-inline-gallery-pagination inset></bento-inline-gallery-pagination>
+  </bento-inline-gallery>
+
+</body>
+```
+
+[/example]
+
+#### Interactivity and API usage
+
+Bento enabled components in standalone use are highly interactive through their API, however the Bento Inline Gallery component does not have any actions of its own. It composes with the Bento Base Carousel component and synchronizes state across the carousel, thumbnails, and pagination indicator. Hence, to interact imperatively with this component, [please use the API provided by the Bento Base Carousel component](https://github.com/ampproject/amphtml/blob/main/extensions/amp-base-carousel/1.0/README.md#interactivity-and-api-usage)
+
+```html
+  <body>
+    <bento-inline-gallery id="inline-gallery">
+      <bento-inline-gallery-thumbnails aspect-ration="1.5" loop></bento-inline-gallery-thumbnails>
+      <bento-base-carousel snap-align="center" visible-count="1.2" loop>
+        <img src="server.com/static/inline-examples/images/image1.jpg">
+        <img src="server.com/static/inline-examples/images/image2.jpg">
+        <img src="server.com/static/inline-examples/images/image3.jpg">
+      </bento-base-carousel>
+      <bento-inline-gallery-pagination inset></bento-inline-gallery-pagination>
+    </bento-inline-gallery>
+  </body>
+  <script>
+    (async () => {
+      await customElements.whenDefined('bento-base-carousel')
+      const carousel = document.querySelector('bento-base-carousel');
+      const carouselApi = carousel.getApi()
+      // automatically updates thumbnails and pagination accordingly
+      carouselApi.next()
+      carouselApi.prev()
+      carouselApi.goToSlide(2)
+
+      // carousel emits `slideChange` events when current slide changes
+      carousel.addEventListener('slideChange', (e) => {
+        console.log(`Moved to slide ${e.data.index}`)
+      })
+    })()
+  </script>
+```
+
+##### Actions
+
+The Bento Inline Gallery component does not support any actions. See above for example on how to interact with with component programmatically.
+
+##### Events
+
+The Bento Inline Gallery component does not support any actions. See above for example on how to interact with with component programmatically.
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-accordion-1.0.css">
```

Comment:
```suggestion
<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-inline-gallery-1.0.css">
```

@alanorozco do you know if the css files should be renamed on release for `bento-` binaries?